taqueria-plugin-flextesa/index.ts

```diff
@@ -49,6 +49,57 @@ Plugin.create(_i18n => ({
             ],
             encoding: "json"
         }),
+        Task.create({
+            task: "typecheck",
+            command: "typecheck <sandboxName> [sourceFiles...]",
```

Comment:
Coming from the UX standpoint here:
1. Can you provide some use cases for when a user would be using the sandboxName optional parameter?
2. Why is [sourceFiles] a required parameter? Can it not default to type checking all files in the configured artifacts directory? (This would be a more intuitive ux)
3. Why does the handler not start the default sandbox if it's not running rather than returning an error message to the user?

Reply:
Angle brackets means it's required and square bracket means it's optional. So sandbox is required, whereas source files are not. In the absence of source files, it will typecheck all source files.

I wanted to let the users start the sandbox. The error message will prompt them to start one. It also makes the implementation simpler for now.

Reply:
My bad on the brackets, however, in the command output it does show both as being required right now: 
```
Positionals:
  sandboxName  The name of the sandbox to use                         [required]
  sourceFile   The name of the Michelson contract you wish to simulate, separate
               d by space                                             [required]
```

Reply:
What is the argument for having a user specify the sandbox?

Is there a circumstance where the user may want to use a sandbox other than local? I can see the need for protocol related type-checking perhaps, but coupling the sandbox this tightly to the type-checker seems like an anti-pattern to me and makes the ux much less intuitive

Reply:
I believe this is for simulation. For typechecking, sourceFile is optional.

The user should be able to just use any sandbox and not have to specify one. But I made it mandatory for now due to command parsing limitations in the CLI.